aptos-move/framework/aptos-framework/sources/vesting_without_staking.spec.move

```diff
@@ -77,7 +77,15 @@ spec aptos_framework::vesting_without_staking {
         include VestingContractActive;
         let vesting_contract_pre = borrow_global<VestingContract>(contract_address);
         let post vesting_contract_post = borrow_global<VestingContract>(contract_address);
-        // ensures vesting_contract.vesting_schedule.start_timestamp_secs <= time::current_timestamp() ==> vesting_contract_pre == vesting_contract_post;
+        let vesting_schedule = vesting_contract_pre.vesting_schedule;
+        let last_vested_period = vesting_schedule.last_vested_period;
+        let next_period_to_vest = last_vested_period + 1;
+        let last_completed_period =
+            (timestamp::spec_now_seconds() - vesting_schedule.start_timestamp_secs) / vesting_schedule.period_duration;
```

Comment:
I think we need to ensure that subtraction is valid (i.e., `timestamp::spec_now_seconds() > vesting_schedule.start_timestamp_secs)

Reply:
I agree, perhaps I need to try to verify distribute first as vest call distribute in the end.